charts/ratify/values.yaml

```diff
@@ -60,7 +60,7 @@ azureManagedIdentity:
 azurekeyvault:
   enabled: false
   vaultURI:
-  tenantID:
+  tenantId:
```

Comment:
is this casing consistent with ClientId?

Reply:
Yes this is consistent casing for the clientId in the values file. The existing tenantId under akvCertConfig also was the same casing. The CRD fields use a different casing of ID though but I thought we should remain consistent with other fields in values.yaml